test/regression/cases/uds_dogstatsd_to_api_nodist/lading/lading.yaml

```diff
@@ -0,0 +1,34 @@
+generator:
+  - unix_datagram:
+      seed: [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53,
+             59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127, 131]
+      path: "/tmp/dsd.socket"
+      variant:
+        dogstatsd:
+          context_range: 1000-8000
+          name_length_range: 32-128
+          tag_key_length_range: 1-128
+          tag_value_length_range: 1:128
```

Comment:
This format is either wrong or surprising

Reply:
Hmm I also find that surprising.

Reply:
Ah, these keys are incorrect: https://github.com/DataDog/lading/blob/9d604dbcf96a0d6514930ac35f18feeb4311a278/lading_payload/src/dogstatsd.rs#L123